README.md

```diff
@@ -90,7 +90,7 @@ Options:
   -c, --comply {fsct3-min,ntia}
                         Compliance standards to check against; see below for details [default: ntia]
   --skip-validation     Skip validation
-  --output {html,json,print,quiet}
+  -r, --output {html,json,print,quiet}
```

Comment:
Is this a breaking change too?

Reply:
People who use --output will still be able to use it without the need to change anything, so it's not a breaking change.

Reply:
Ah, great. Thank you.